src/main/java/org/sagebionetworks/bridge/udd/synapse/SynapseHelper.java

```diff
@@ -61,6 +63,8 @@ public final void setSynapseClient(SynapseClient synapseClient) {
      * @throws SynapseException
      *         if calling Synapse fails
      */
+    @RetryOnFailure(attempts = 5, delay = 100, unit = TimeUnit.MILLISECONDS, types = SynapseException.class,
+            randomize = false)
```

Comment:
Without exponential back-off, should the delay be larger? If you're being throttled, 5 tries in 500 ms seems like it would not get around this.